.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: tests
+
+on: [push, pull_request]
+
+env:
+  PYTHONHASHSEED: 1042466059
+
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [2.7, pypy2, pypy3, 3.6, 3.7, 3.8, 3.9]
```

Comment:
These same versions need listed in the classifiers in setup.py and in the interpreter list in tox.ini. Plus a note in CHANGES.rst about the expanded support.

Reply:
As we are only expanding support is a minor version bump fine?

Reply:
I think so, that’s what zope does.